packages/react-instantsearch/src/connectors/connectMultiRange.js

```diff
@@ -36,46 +36,31 @@ function getCurrentRefinement(props, state) {
   return '';
 }
 
+/**
+ * MultiRange connector provides the logic to build a widget that will
+ * give the user the ability to select a range value for a numeric attribute.
+ * Ranges are defined statically.
+ * @name MultiRange
+ * @kind HOC
+ * @category connector
```

Comment:
Why do we have to say its both a connector and a HOC? Has this any impact on the documentation/website?